elixir/apps/web/lib/web/live/sites/new_token.ex

```diff
@@ -236,20 +257,29 @@ defmodule Web.Sites.NewToken do
             echo "Unsupported architecture"; \\
             exit 1 ;; \\
         esac; \\
-        wget -O /usr/local/bin/firezone-gateway $bin_url; \\
-        chmod +x /usr/local/bin/firezone-gateway; \\
-        mkdir -p /etc/firezone; \\
-        chmod 0755 /etc/firezone; \\
-        iptables-nft -A FORWARD -i tun-firezone -j ACCEPT; \\
-        iptables-nft -A FORWARD -o tun-firezone -j ACCEPT; \\
-        iptables-nft -t nat -A POSTROUTING -o e+ -j MASQUERADE; \\
-        ip6tables-nft -A FORWARD -i tun-firezone -j ACCEPT; \\
-        ip6tables-nft -A FORWARD -o tun-firezone -j ACCEPT; \\
-        ip6tables-nft -t nat -A POSTROUTING -o e+ -j MASQUERADE; \\
+        curl -Ls $bin_url -o /usr/local/bin/firezone-gateway; \\
+        chgrp firezone /usr/local/bin/firezone-gateway; \\
+        chmod 0750 /usr/local/bin/firezone-gateway; \\
+        setcap 'cap_net_admin+eip' /usr/local/bin/firezone-gateway; \\
       fi; \\
+      groupadd -f firezone; \\
+      id -u firezone &>/dev/null || useradd -r -g firezone -s /sbin/nologin firezone; \\
```

Comment:
`useradd` did not work for me as part of the unitfile, will need to test it manually

Reply:
Had the user create in an explicit step in the commit before this one. will revert to that commit and test it.

Reply:
Updated.